plugins/inputs/dns/README.md

```diff
@@ -0,0 +1,51 @@
+# DNS Query Input Plugin
+
+The DNS plugin gathers dns query times in miliseconds - like [Dig](https://en.wikipedia.org/wiki/Dig_\(command\))
+
+### Configuration:
+
+```
+# Sample Config:
+[[inputs.dns_query]]
+  ### Domains or subdomains to query
+  domains = ["mjasion.pl"] # required
```

Comment:
@sparrc Well, I dont remember to use `dig` without any domain ;-) Yes, its possible
`dig` manual says: 
>  When no command line arguments or options are given, dig will perform an NS query for "." (the root).
I've changed default record type to `NS`

---
I also have checked another corner case: what record type is used, if for example I am quering record `MX` of domain `.`(root).  Dns server returns `SOA` response. 
I looked in query logs on server and `dig` was quering for `MX` :-)